pkg/index/index.go

```diff
@@ -54,6 +53,7 @@ func New(rewriters []PathRewriter) *State {
 		shardWorkspaceName:        map[string]map[logicalcluster.Name]string{},
 		shardClusterParentCluster: map[string]map[logicalcluster.Name]logicalcluster.Name{},
 		shardBaseURLs:             map[string]string{},
+		workspaceNameMount:        map[logicalcluster.Name]map[string]string{},
```

Comment:
what are the two strings?

Reply:
`Cluster, workspace, serialized-to-string-mount-point-object`

Reply:
clusterWorkspaceMountAnnotation ?